lib/components/screens/retweeters/retweets/cubit/retweets_cubit.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:bloc/bloc.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:dart_twitter_api/twitter_api.dart';
+import 'package:harpy/api/api.dart';
+import 'package:harpy/components/components.dart';
+import 'package:harpy/core/core.dart';
+
+class RetweetersCubit extends Cubit<PaginatedState<BuiltList<UserData>>>
+    with RequestLock, HarpyLogger {
+  RetweetersCubit({
+    required this.tweetId,
+  }) : super(const PaginatedState.loading());
+
+  final String tweetId;
+
+  Future<void> _request({bool clearPrevious = false}) async {
+    if (clearPrevious) {
+      emit(const PaginatedState.loading());
+    }
+
+    final users = await app<TwitterApi>()
+        .tweetService
+        .retweets(id: tweetId, count: 100)
+        .handleError(twitterApiErrorHandler)
+        .then(
+          createUsers,
+        );
```

Comment:
```suggestion
    final users = await app<TwitterApi>()
        .tweetService
        .retweets(id: tweetId, count: 100)
        .then((tweets) => tweets.map((tweet) => UserData.fromUser(tweet.user)))
        .handleError(twitterApiErrorHandler);
```

We should handle errors when mapping the data.
Also, this way we can easily check whether an error occurred by checking if `users` is `null` or whether no retweeters exist by checking whether `users` is empty.